net/tstun/wrap.go

```diff
@@ -522,7 +523,9 @@ type natV4Config struct {
 	// dstMasqAddrs is map of dst addresses to their respective MasqueradeAsIP
 	// addresses. The MasqueradeAsIP address is the address that should be used
 	// as the source address for packets to dst.
-	dstMasqAddrs views.Map[netip.Addr, netip.Addr] // dst -> masqAddr
+	dstMasqAddrs views.Map[key.NodePublic, netip.Addr] // dst -> masqAddr
+
+	routingTable *table.RoutingTable
```

Comment:
Add a comment describing the key and value, as well as saying which ip (native ip, masq ip etc) it stores as value

Reply:
renamed to `dstAddrToPeerKeyMapper`